etc/adminhtml/system/global.xml

```diff
@@ -48,5 +48,12 @@
 				<field id="test_mode">1</field>
 			</depends>
 		</field>
+
+		<field id="active" showInDefault="1" showInStore="1" showInWebsite="1" sortOrder="50" translate="label" type="select">
+			<label>Send Mail Change Order Status</label>
+			<comment><![CDATA[Send e-mail every time when a order change status - (just Mundipagg status). <p><strong style="color:red">Warning!</strong> If enabled, make sure you have set up your mail server correctly, otherwise you will run the risk of slowdowns and crashes.</p>]]></comment>
```

Comment:
Mudar texto para: 
Send an e-mail every time that an order status is changed by a Mundipagg's webhook

Make sure you have set up your mail server correctly before enable it, otherwise you will run the risk of slowdowns and crashes of your platform.

Reply:
Esses textos precisam ter seu equivalente em português, ou seja, possuir tradução.